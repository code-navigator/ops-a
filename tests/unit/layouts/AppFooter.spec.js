import AppFooter from '@/components/layouts/AppFooter/'
import Vue from 'vue'
import Vuetify from 'vuetify'
import { shallowMount } from '@vue/test-utils'
import { TestHelpers } from '../TestHelpers'

Vue.use(Vuetify)

describe('layout: AppFooter.vue', () => {
  const name = 'AppFooter'
  let h, wrapper

  // Run these statements before each test
  beforeEach(() => {
    wrapper = shallowMount(AppFooter)

    h = new TestHelpers(wrapper, expect)
  })

  // Is component mounted?
  test('should mount without errors', () => {
    h.isInstance()
  })

  // Does component have correct name?
  test('has name', () => {
    h.hasName(name)
  })

  // Does component render correct template
  test('renders correctly', () => {
    h.renders()
  })
})
