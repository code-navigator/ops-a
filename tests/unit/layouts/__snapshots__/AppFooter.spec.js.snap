// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`layout: AppFooter.vue renders correctly 1`] = `<v-footer-stub height="32" app="app" color="primary" dark="dark"><span class="white--text">Â© 2018</span></v-footer-stub>`;
